src/sphinx.md

```diff
@@ -298,3 +311,7 @@ Constructing a reply packet from a SURB is straightforward:
 
 The constructed packet should be sent to the mixnode with index `first_mixnode_index`. The session
 is implicit: it should match the session used by the request message containing the SURB.
+
+
+// TODO generally I don't know if all algo construction are strictly necessary, can certainly help,
+// I did not really check them though.
```

Comment:
Yeah I agree a bunch of this stuff is probably not really necessary as it can be inferred from the rest of the spec. Writing it seemed worthwhile to me at the time...

Reply:
Probably better to keep it then (I already had an idea of how things work when reading).